client/cli/src/arg_enums.rs

```diff
@@ -212,6 +214,8 @@ impl std::str::FromStr for Database {
 		if s.eq_ignore_ascii_case("rocksdb") {
 			Ok(Self::RocksDb)
 		} else if s.eq_ignore_ascii_case("paritydb-experimental") {
+			Ok(Self::ParityDbDeprecated)
```

Comment:
This function gets called multiple times, so can't just print a warning here.

Reply:
We could only log it once using `OnceCell` for example